product/utilities/containers/index.md

```diff
@@ -40,3 +40,28 @@ layout: page
 	<p>100%</p>
 </div>
 {% endhighlight %}
+
+
+
+<hr />
+
+## Bootstrap Containers
+
+For the grid system, use <code>.container</code> for making rows with a responsive fixed width or <code>.container-fluid</code> for a full width container that spans the entire width of your viewport.
+
+<div class="container bg-c-g200 p-4 m-bottom-4 u-border-radius">
+	<p><strong>.container</strong> (fixed)</p>
```

Comment:
While you're here, can you add `class="m-0"` to each of the `<p>` elements with the container text? The spacing on each of the blocks is a bit uneven on the bottom due to bottom margin on each paragraph. The `m-0` class will override that and set all margin to 0.